README.md

```diff
@@ -65,25 +67,25 @@ as an interactive `Flame Graph` that can be viewed in a browser.
 
 ## Basic usage
 
-* [Getting Started](docs/GettingStarted.md)
-* [Profiler Options](docs/ProfilerOptions.md)
-* [Profiling Modes](docs/ProfilingModes.md)
-* [Integrating async-profiler](docs/IntegratingAsyncProfiler.md)
-* [Profiling In Container](docs/ProfilingInContainer.md)
+- [Getting Started](docs/GettingStarted.md)
```

Comment:
If there is a list with a sublist, is prettier smart enough not to use the same delimiter for both?

Reply:
Not sure I follow, sublist will be indented:

```
- Item
  - Subitem
```

- Item
  - Subitem

Reply:
I mean
```
 - Top Item 1
   * subitem 1.1
   * subitem 1.2
 - Top Item 2
```
I don't want autoformatting to replace `*` with `-` in this case.